src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5339,4 +5339,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_OutVariableCannotBeByRef" xml:space="preserve">
     <value>An out variable cannot be declared as a ref local</value>
   </data>
+  <data name="WRN_InvalidSeverityInAnalyzerConfig" xml:space="preserve">
+    <value>The diagnostic '{0}' was given an invalid severity '{1}' in analyzer config file.</value>
```

Comment:
>in analyzer config file [](start = 68, length = 23)

"in analyzer config file"? In THE analyzer config file? In AN analyzer config file? Can we identify the offending config file?